renderer/shared/providers/identity-context.js

```diff
@@ -57,6 +61,11 @@ function IdentityProvider({children}) {
     fetchData()
   }, 1000 * 1)
 
+  useInterval(
```

Comment:
maybe usePoll ?

Reply:
unfortunately it's now compatible only with `useRpc`